contrib/openbsm/libbsm/bsm_audit.c

```diff
@@ -215,12 +215,12 @@ au_write(int d, token_t *tok)
  * XXX: Assumes there is sufficient space for the header and trailer.
  */
 static int
-au_assemble(au_record_t *rec, short event)
+au_assemble(au_record_t *rec, short event, struct timeval *tm)
 {
 #ifdef HAVE_AUDIT_SYSCALLS
 	struct in6_addr *aptr;
 	struct auditinfo_addr aia;
-	struct timeval tm;
+	struct timeval localtm;
```

Comment:
I wouldn't change the local variable name but use another name for an argument, let's say eventtm.